Backend/src/test/java/ktsnvt/tim1/services/LocationServiceIntegrationTests.java

```diff
@@ -0,0 +1,275 @@
+package ktsnvt.tim1.services;
+
+import ktsnvt.tim1.DTOs.LocationDTO;
+import ktsnvt.tim1.DTOs.SeatGroupDTO;
+import ktsnvt.tim1.exceptions.EntityNotFoundException;
+import ktsnvt.tim1.exceptions.EntityNotValidException;
+import ktsnvt.tim1.model.Location;
+import ktsnvt.tim1.model.Seat;
+import ktsnvt.tim1.model.SeatGroup;
+import ktsnvt.tim1.repositories.LocationRepository;
+import org.hibernate.id.GUIDGenerator;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.test.annotation.Rollback;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import javax.persistence.ExcludeDefaultListeners;
+import javax.transaction.Transactional;
+import java.util.HashSet;
+import java.util.Optional;
+
+import static org.junit.Assert.*;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ActiveProfiles("test")
+public class LocationServiceIntegrationTests {
+    @Autowired
+    private LocationService locationService;
+
+    @Autowired
+    private LocationRepository locationRepository;
+
+    @Test
+    void getLocations_pageRequestSent_returnedLocations() {
```

Comment:
This method should be named getLocations_pageRequestSent_locationsReturned to be consistent with names of other test methods.

Reply:
Changes made on other branch